src/Lecture4.hs

```diff
@@ -223,7 +285,10 @@ implement the next task.
 -}
 
 combineRows :: NonEmpty Row -> Stats
-combineRows = error "TODO"
+combineRows = combineStats . fmap rowToStats
+
+combineStats :: NonEmpty Stats -> Stats
+combineStats (a :| as) = foldl' (<>) a as
```

Comment:
Removed my recursive go with bang in favor of `foldl'`. Looks identical from the resource usage pov.